girder/images/images/image.py

```diff
@@ -0,0 +1,105 @@
+import cherrypy
+
+from girder.api import access
+from girder.api.describe import Description, autoDescribeRoute
+from girder.api.rest import Resource, RestException
+from girder.constants import AccessType, SortDir, TokenScope
+
+from images.models.image import Image as ImageModel
+from images.models.image import ImageTypes
+
+
+class Image(Resource):
+
+    def __init__(self):
+        super(Image, self).__init__()
+        self.resourceName = 'images'
+        self.route('GET', (), self.find)
+        self.route('GET', ('unique', ), self.find_unique)
```

Comment:
Does this need to be a separate endpoint? i seems that the endpoint above could be used if a extra parameter was provide such as `unique=true`?

Reply:
Sure, I can put it in the same endpoint. The only reason I separated it is because the code and logic are much more complicated for `unique=true`, since the unique case involves aggregates and extra sorting.

But I think I can give them the same rest endpoint and then use separate functions for some of the logic.

Reply:
Yes, I am not say we should combine the logic, just expose it through a single endpoint.

Reply:
This is now exposed through a single endpoint, with a "unique" parameter to indicate whether to call the unique function or not.